env.js

```diff
@@ -1,13 +1,14 @@
 var log = require('loglevel');
 
 module.exports = function() {
-  if (process.env.COUCH2PG_DEBUG) {
+  if (process.env.COUCH2PG_DEBUG || true) {
     log.setDefaultLevel('debug');
   } else {
     log.setDefaultLevel('info');
   }
 
   return {
+    legacyMode: process.env.LEGACY_MODE || false,
```

Comment:
I'm not a fan of the name `legacyMode`. It'll become difficult once 2.6 is considered legacy. Can we give it a name that represents the feature we're interested in? ie: What's the real difference between legacy mode and non-legacy mode? For bonus points, can we detect this in the db rather than having to provide it?